.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,259 @@
+
+## Picking an open issue
+
+If you want to pick an open issue, please mention that in the issue, and it will be assigned to you. This will prevent the possible double work by different people on the same issue.
+
+## Contributing to the project
+
+If you want to contribute to this project, fork the repo and open a pull request to the `develop` branch. If you have push access to this repo, you can create a feature branch and then open an intra-repo PR from that branch to the `develop`.
+
+# Considerations when writing sniffs
+
+## Unit Testing
+
+All PRs which affect sniffs, whether bug fixes to existing sniffs, or the addition of a new sniff, should be accompanied by unit tests.
+
+### Pre-requisites
+
+* PHP 5.4 or higher
+* WordPress-Coding-Standards 1.0.0 or higher
+* PHP_CodeSniffer 3.3.0 or higher
+* PHPUnit 4.x, 5.x, 6.x or 7.x
+* PHPCompatibility 8.0 or higher
+
+### Composer setup
+
+Use Composer to install all the necessary dependencies to write and run unit tests. Running
+
+```bash
+composer install
+```
+
+from the root of the cloned repository, will install `PHP_CodeSniffer`, `WordPress Coding Standards`, `PHPUnit`, `PHPCompatibility` and `security advisories` which ensures that your application doesn't have installed dependencies with known security vulnerabilities.
+
+### Other setups
+
+If you have PHP_CodeSniffer (PHPCS) and/or WordPress Coding Standards (WordPressCS) already installed, for instance as git clones, because you either contribute to these libraries as well or you want to develop for or test WPThemeReview with bleeding edge versions of either PHP_CodeSniffer or WordPress Coding Standards, you need to take some additional steps to make it all work.
+
+First, make sure you also have PHPCompatibility and make sure the `installed_paths` for PHP_CodeSniffer is setup correctly.
+
+You can see how this can be done by reading the official PHPCS [documentation](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#setting-the-installed-standard-paths).
+
+#### Method 1
+
+Set up global environment variables to point to PHP_CodeSniffer and WordPress Coding Standards:
+
+```bash
+PHPCS_DIR: I:/path/to/PHP_CodeSniffer/
+WPCS_DIR: I:/path/to/WordPressCS/
+```
+
+If you do that, the unit tests should be able to work correctly.
+
+#### Method 2
+
+If you do not want to set up environment variables on your system
+
+* Add a file called `.pathtowpcs` to the project root. The only content in that file should be the full absolute path to your WordPressCS install.
+* Copy the `phpunit.xml.dist` file, rename the copied file `phpunit.xml` and make sure it's in the project root.
+Now add the following to that file, adjusting the paths to reflect those on your system:
+
+```xml
+  <php>
+    <env name="PHPCS_DIR" value="path/to/PHP_CodeSniffer/"/>
+    <env name="WPCS_DIR" value="path/to/WordPressCS/"/>
+  </php>
+```
+
+(and don't remove the existing line within the `<php>` block.)
+
```

Comment:
Add: `Once you've done that, both running the sniffs as well as the unit tests should work correctly.`